packages/cli/src/controller/deploy-controller.spec.ts

```diff
@@ -53,13 +52,13 @@ async function deployTestProject(
     url
   );
 
-  const endpoint = await networkEndpoints(url);
+  const endpoint = 'wss://polkadot.api.onfinality.io/public-ws';
```

Comment:
I don't think we need to default to polkadot. Chances are slim that's the network being used